src/beneuro_data/conversion/anipose_interface.py

```diff
@@ -120,6 +155,10 @@ def load_anipose_from_h5(self) -> np.ndarray:
 
         return pose_data
```

Comment:
Do we want to keep this? Are there any sessions where you have HDF5 instead of CSV?
Maybe add a deprecation warning if it's called for whatever reason, and we can delete it later?

Reply:
Added a warning